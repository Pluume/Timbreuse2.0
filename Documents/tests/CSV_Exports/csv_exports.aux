\relax 
\providecommand\hyper@newdestlabel[2]{}
\catcode `:\active 
\catcode `;\active 
\catcode `!\active 
\catcode `?\active 
\bibstyle{biblatex}
\bibdata{csv_exports-blx}
\citation{biblatex-control}
\abx@aux@sortscheme{nty}
\abx@aux@refcontext{nty/global/}
\providecommand\HyperFirstAtBeginDocument{\AtBeginDocument}
\HyperFirstAtBeginDocument{\ifx\hyper@anchor\@undefined
\global\let\oldcontentsline\contentsline
\gdef\contentsline#1#2#3#4{\oldcontentsline{#1}{#2}{#3}}
\global\let\oldnewlabel\newlabel
\gdef\newlabel#1#2{\newlabelxx{#1}#2}
\gdef\newlabelxx#1#2#3#4#5#6{\oldnewlabel{#1}{{#2}{#3}}}
\AtEndDocument{\ifx\hyper@anchor\@undefined
\let\contentsline\oldcontentsline
\let\newlabel\oldnewlabel
\fi}
\fi}
\global\let\hyper@last\relax 
\gdef\HyperFirstAtBeginDocument#1{#1}
\providecommand\HyField@AuxAddToFields[1]{}
\providecommand\HyField@AuxAddToCoFields[2]{}
\@writefile{toc}{\boolfalse {citerequest}\boolfalse {citetracker}\boolfalse {pagetracker}\boolfalse {backtracker}\relax }
\@writefile{lof}{\boolfalse {citerequest}\boolfalse {citetracker}\boolfalse {pagetracker}\boolfalse {backtracker}\relax }
\@writefile{lot}{\boolfalse {citerequest}\boolfalse {citetracker}\boolfalse {pagetracker}\boolfalse {backtracker}\relax }
\select@language{french}
\@writefile{toc}{\select@language{french}}
\@writefile{lof}{\select@language{french}}
\@writefile{lot}{\select@language{french}}
\@writefile{toc}{\contentsline {section}{\numberline {1}Introduction}{1}{section.1}}
\@writefile{toc}{\contentsline {section}{\numberline {2}R\IeC {\'e}sultats attendus}{1}{section.2}}
\@writefile{toc}{\contentsline {section}{\numberline {3}Proc\IeC {\'e}dure de test}{1}{section.3}}
\@writefile{toc}{\contentsline {section}{\numberline {4}R\IeC {\'e}sultats obtenus}{1}{section.4}}
\@writefile{toc}{\contentsline {section}{\numberline {5}Conclusion}{1}{section.5}}
\newlabel{LastPage}{{}{1}{}{page.1}{}}
\xdef\lastpage@lastpage{1}
\xdef\lastpage@lastpageHy{1}
