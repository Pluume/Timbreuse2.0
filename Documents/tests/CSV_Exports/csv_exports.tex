\documentclass[10pt,a4paper,onecolumn]{article}
\usepackage[top=30pt,bottom=30pt,left=48pt,right=46pt]{geometry}
\usepackage{a4wide}
\usepackage{titling}
\setlength{\droptitle}{-5em}
\usepackage[version=3]{mhchem}
\usepackage{siunitx}
\usepackage{graphicx}
\usepackage{amsmath}
\usepackage[utf8]{inputenc}
\usepackage[english,francais]{babel}
\usepackage[backend=bibtex]{biblatex}
\usepackage[pdftex]{hyperref}
\setlength\parindent{0pt}
\usepackage{xcolor}
\hypersetup{colorlinks,urlcolor=blue}
\usepackage{fancyhdr}
\usepackage{lastpage}
\pagestyle{fancy}
\fancyhf{}
\chead{Francis Le Roy}
\lfoot{P1699/MCT}
\cfoot{\today}
\rfoot{Page \thepage \hspace{1pt} of \pageref{LastPage}}
% \bibliography{bib}
\renewcommand{\labelenumi}{\alph{enumi}.}



\title{Rapport de test \\ Exportation des CSV \\ lorsque les élèves timbres}

\author{Francis \textsc{Le Roy}}

\date{\today}

\begin{document}

\maketitle
\thispagestyle{fancy}

\begin{center}
\begin{tabular}{l r}
Date de réalisation : & 2 Mars, 2017 \\
Projet : & P1699 \\
\end{tabular}
\end{center}

\section{Introduction}
Lorsque les professeurs veulent avoir accès au CSV, il doivents passer un tag maître à une timbreuse esclave, celle-ci va l'envoyer au serveur. Le serveur répondra en donnant l'ordre à la timbreuse esclave de copier tous ses CSV sur les supports de stockage amovibles disponibles. Ce test ne vérifie que la fonction exportant les données sur les supports externes.
\section{Résultats attendus}
Un dossier daté devrait être crée sur la clé USB. De plus, tous les CSV de la timbreuse devraient être stockés sur la clé dans le dossier daté.
\section{Procédure de test}
Il faut utiliser le fichier \href{run:../../../test/csv_export.js}{script} Node JS, qui va effectuer la fonction exportant les CSV. Si la fonction s'execute correctement, le message "Done" sera affiché dans la console
\section{Résultats obtenus}
Lorsque le script est executé, tous les CSV sont copiés dans un dossier daté sur la clé USB. Le message "Done" est apparu dans la console.
\section{Conclusion}
Cette fonctionnalité est donc opérationnelle.
% \printbibliography

\end{document}
