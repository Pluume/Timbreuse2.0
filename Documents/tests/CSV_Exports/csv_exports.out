\BOOKMARK [1][-]{section.1}{Introduction}{}% 1
\BOOKMARK [1][-]{section.2}{R\351sultats attendus}{}% 2
\BOOKMARK [1][-]{section.3}{Proc\351dure de test}{}% 3
\BOOKMARK [1][-]{section.4}{R\351sultats obtenus}{}% 4
\BOOKMARK [1][-]{section.5}{Conclusion}{}% 5
